### PR TITLE
DCOS-16274:  Add Intl polyfill

### PR DESCRIPTION
**Pull request by orlandohohmeier**
Add a Intl (ECMA-402) polyfill to fix application failures due to the missing API.

Closes DCOS-16274

### CODE REVIEW

**Comment by nLight** [comment]
I was able to login to OSS Cluster using Firefox 28 that shouldn't have intl api

**Comment by orlandohohmeier** [comment]
Thanks for the review! Will merge once the system tests pass.

**Comment by orlandohohmeier** [comment]
ℹ️  Please note that I changed the base to `release/1.10`

File: src/js/index.js [hunk 1]
```diff
@@ -125,10 +125,23 @@ RequestUtil.json = function(options = {}) {
...
+    ConfigStore.addChangeListener(CONFIG_ERROR, onConfigurationError);
 
-  // Load configuration
-  ConfigStore.fetchConfig();
+    // Load configuration
+    ConfigStore.fetchConfig();
+  }
+
+  if (!global.Intl) {
+    require.ensure(["intl", "intl/locale-data/jsonp/en.js"], function(require) {
```
Comment (weblancaster) [inline src/js/index.js]: we can store `intl/locale-data/jsonp/en.js` and re-use it. not big deal though.

Comment (orlandohohmeier) [inline src/js/index.js]: Good point. It might even makes sense to create a list of needed locals and use that to require the respective changes.

**Pull request merged**